minpy/array.py

```diff
@@ -242,7 +242,7 @@ def __str__(self):
         return str(self._val)
 
     def __repr__(self):
-        return self.__str__()
+        return repr(self._val)
```

Comment:
`repr` is still there. What you pointed out is an alternative implementation of `repr`.